Newsweek Clone

hamdiheb opened the pull request.

hamdiheb committed:
Setup linters

hamdiheb committed:
Setup Top navbar

hamdiheb committed:
Fix css file as linter asked

fortuneonyeka committed:
news week

added the tabs

fortuneonyeka committed:
newsiweek

added the text styles of the nav tabs

fortuneonyeka committed:
news week

added nav tab 2 and styling

hamdiheb committed:
Update css file

hamdiheb committed:
Update css file

hamdiheb committed:
Update css file

hamdiheb committed:
Update css file

fortuneonyeka committed:
news week

corrected on index.html

hamdiheb committed:
Update css file

hamdiheb committed:
Update README file

hamdiheb committed:
Update README file

hamdiheb committed:
Update README file

hamdiheb committed:
Update README file

mricanho reviewed (changes requested):
# 🔴  Changes Required 🔴  

Hello @hamdiheb @fortuneonyeka, you did a good job in this milestone🔥, just a few changes are required:

- [x]  **Your nav is not so good looking right now, remember that you are cloning the NW website, so you need to put extra attention to the details**
- [x] **You need to add a logo in the nav, there are plenty NW logos on the internet that you can use** 
- [x] **Double check the size of the text in the nav, also, align the items**
- [x] **You missed the "Login" in the nav, please add it**
- [x] **The hamburger button only needs to be in the mobile version**
- [x] **The search input and the search icon are not aligned, please fix that**
- [x] **You have an underline in all the links, remove it, and check the transition color for every case**
- [x] **Check the margin and the color of the social icons in the footer**

As described in the Code reviews limit [Policy ](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)you have 2 code reviews left for this project.

![work hard](https://user-images.githubusercontent.com/60631456/113642841-5f8ec000-9646-11eb-8ec2-ff7ba39a110c.gif)

You can contact me on slack if you have any questions just search for me as @ Miguel Ricaño.

### 💻 👩🏽‍💻  Happy Coding!👨🏽‍💻  🖥️

hamdiheb committed:
Update logo top nav

fortuneonyeka committed:
news-week

update the navbar

hamdiheb committed:
Update navbar

fortuneonyeka committed:
news-week

update navlinks

hamdiheb committed:
Update navbar

hamdiheb committed:
Update css file as linter want

hamdiheb committed:
Update css file as linter want

fortuneonyeka committed:
news-week

update social icons with margin,made the foot log a link and removed the underlines from the footer links

fortuneonyeka committed:
news-week

made the social icons a link,added hover effect and color

fortuneonyeka committed:
news-week

added border to the navbar

hamdiheb committed:
Update css file for social media icons

smcommits commented on css/index.css:
```diff
@@ -0,0 +1,172 @@
+  }
+
+  .search_icon {
+    display: flex;
+    float: right;
+    position: relative;
```
You should use bootstrap classes for these.

smcommits commented on css/index.css:
```diff
@@ -0,0 +1,172 @@
+  }
+
+  .nav-fast-link {
+    display: flex;
```
You should use bootstrap classes for this as well.

hamdiheb replied:
Thank you so much for approving our project , and thank you so much for this advice it's really helpful

smcommits reviewed (approved):
# Status: Approved :+1: 

## Hello, @hamdiheb and @fortuneonyeka! You guys have done a great job. The components look great. Nice work implementing the changes that the previous TSE required. This PR is now approved. You can go ahead and merge the branch. 

## Optional

## Please note that the following suggestion is optional but highly recommended. 

- [ ] **You should use more bootstrap classes. Please refer to the inline comments**

There are plenty of declarations in your CSS that can be easily achieved with Bootstrap. Not using it defeats the purpose of using a framework. I have highlighted some of the rules in the inline comments.
